src/functions/delimsizing.js

```diff
@@ -38,8 +38,8 @@ const delimiters = [
     "(", ")", "[", "\\lbrack", "]", "\\rbrack",
     "\\{", "\\lbrace", "\\}", "\\rbrace",
     "\\lfloor", "\\rfloor", "\\lceil", "\\rceil",
-    "<", ">", "\\langle", "\\rangle", "\\lt", "\\gt",
-    "\\lvert", "\\rvert", "\\lVert", "\\rVert",
+    "<", ">", "\\langle", "\u27e8", "\\rangle", "\u27e9", "\\lt", "\\gt",
+    "\\lvert", "\\rvert", "\u2223", "\\lVert", "\\rVert", "\u2225",
```

Comment:
Just out of curiosity, is it valid to have right delimiters, e.g., `\rangle`, as a left delimiter?

Reply:
I checked with quicklatex.com and it appears so although I'm not sure in what context you'd actually want to do that.